src/app/services/sms/sms.utils.ts

```diff
@@ -0,0 +1,28 @@
+import { smsConfig } from '../../config/features/sms.config'
+
+export enum smsThreshold {
+  Half = 0.5,
+  ThreeQuarters = 0.75,
+}
+
+/**
+ *
+ * @param initialCount smsCount before a db update
+ * @returns
+ */
+export const hasHitSmsThreshold = ({
+  smsCount,
+}: {
+  smsCount: number
+}): smsThreshold | false => {
+  if (
+    smsCount === Math.floor(smsConfig.smsVerificationLimit * smsThreshold.Half)
+  )
+    return smsThreshold.Half
+  else if (
+    smsCount ===
+    Math.floor(smsConfig.smsVerificationLimit * smsThreshold.ThreeQuarters)
+  )
+    return smsThreshold.ThreeQuarters
+  return false
```

Comment:
/nit would prefer using guard clause patterns whenever possible rather than sequential `if-else`s. I generally also try to avoid using `else`.

```suggestion
  if (
    smsCount === Math.floor(smsConfig.smsVerificationLimit * smsThreshold.Half)
  ) {
    return smsThreshold.Half
  }
  if (
    smsCount ===
    Math.floor(smsConfig.smsVerificationLimit * smsThreshold.ThreeQuarters)
  ) {
    return smsThreshold.ThreeQuarters
  }
  return false
```